build/docker/debian.Dockerfile

```diff
@@ -81,5 +81,26 @@ RUN apt -y update && apt -y upgrade && apt -y install openjdk-11-jre \
 
 RUN dotnet --version
 
+RUN apt -y update && apt -y upgrade && apt -y install \
+    php \
+    php-json \
+    php-openssl \
+    php-curl \
+    php-dom \
+    php-mbstring \
+    php-xml \
+    php-phar \
+    php-tokenizer \
+    php-xmlwriter \
+    php-simplexml \
+    php-xmlreader \
+    php-session \
+    php-ctype && \
+    apt -y clean && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
+
+RUN php -v && composer --version
+
```

Comment:
If you look in your checks, something is not working here. I think it may be that you need to add the PHP repository or something.

Reply:
@4ernovm please use https://computingforgeeks.com/how-to-install-php-8-2-on-debian/

where we explicitly install PHP 8.2 :)

Reply:
see also my comment on alpine image for which extensions to install